### PR TITLE
Added a reference to example config file.

### PR DESCRIPTION
**Pull request by xerophileshark**
When running `djehuty web`, I had difficulty figuring out how to properly set `DJEHUTY_CONFIG_FILE`. I thought it might be helpful to prevent such a case for new people trying to run the project if you reference the example xml config file in the README.md.

### CODE REVIEW

**Comment by roelj** [comment]
Hi @xerophileshark . Thank you a lot for looking into Djehuty and improving the README.
One small question: Is it OK if we rephrase it to:
```
An example of a configuration file can be found in [etc/djehuty/djehuty-example-config.xml](./etc/djehuty/djehuty-example-config.xml).
```

(So, add "a")
There's no need to redo the pull request; If you agree I can modify it in-place. :-)

Then, finally, I'd like to merge it by signing the commit. That means you are the author, and I use my PGP key to sign it. This is equivalent to [the commits of Madeleine](https://github.com/4TUResearchData/djehuty/commits?author=mdesmaele).
Could you let me know whether the author information is correct (specifically looking at the e-mail address :-)).

Is that OK for you? :-)

Thanks again for your efforts!

**Comment by xerophileshark** [comment]
Hi @roelj,

Thank you for your reply.
Sure, you can rephrase and modify it as you wish yourself.

I used the ID-based GitHub-provided [noreply email](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username#your-git-commits): `30368346+xerophileshark@users.noreply.github.com` to sign my commit. I believe the PGP for this email is setup properly in my account. Please let me know if this is a problem.

Have a nice evening!

**Comment by xerophileshark** [comment]
Hi @roelj. Yes, I will use this email for my future commits (I will not change this email to keep my primary email private).

**Comment by roelj** [comment]
Hi @xerophileshark . Cool. Thank you a lot for your effort! I merged it in [a561ad4](https://github.com/4TUResearchData/djehuty/commit/a561ad45f67e27791d764d43eaa6d7958e23d6bf).
